ci/scripts/check_bazel_deprecated_tags.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
```

Comment:
wouldn't it be better to only check for supported tags, i.e., an allow list instead of a deny list?

Reply:
I actually just whipped it up to demonstrate that I'd eliminated all the "vivado" tags. I intend to delete it in a follow up PR :)

Reply:
(Although it turns out I forgot to make the script executable. Just force pushed a new patchset and I'll let the CI run.)